gui/src/Layout.js

```diff
@@ -97,26 +104,23 @@ class Layout extends Component {
           onClick={this.toggleDrawer}
         >
           <List className={classes.drawerList}>
-            <ListItem button>
-              <Link to='/' className={classes.menuitem}>
-                <ListItemText primary='Jobs' />
-              </Link>
+            <ListItem button component={Link} to='/' className={classes.menuitem}>
+              <ListItemText primary='Jobs' />
+            </ListItem>
+            <ListItem button component={Link} to='/bridges' className={classes.menuitem}>
+              <ListItemText primary='Bridges' />
             </ListItem>
-            <ListItem button>
-              <Link to='/bridges' className={classes.menuitem}>
-                <ListItemText primary='Bridges' />
-              </Link>
+            <ListItem button component={Link} to='/config' className={classes.menuitem}>
+              <ListItemText primary='Configuration' />
             </ListItem>
-            <ListItem button>
-              <Link to='/config' className={classes.menuitem}>
-                <ListItemText primary='Configuration' />
-              </Link>
+            <ListItem button component={Link} to='/about' className={classes.menuitem}>
+              <ListItemText primary='About' />
             </ListItem>
-            <ListItem button>
-              <Link to='/about' className={classes.menuitem}>
-                <ListItemText primary='About' />
-              </Link>
+            { this.props.authenticated &&
+            <ListItem button onClick={this.signOut} className={classes.menuitem}>
+              <ListItemText primary='Sign Out' />
```

Comment:
Maybe we want to add a sign out button next to the drawer icon?

Reply:
That's a good idea. Let's save that for another PR.